src/care_radiology/api/webhooks.py

```
@@ -114,3 +135,173 @@ def save_webhook(self, request):
+                cache.set("admin_access_token", access_token, 600)
+                logger.info("[MPPS] New access token obtained and cached")
+            else:
+                logger.info("[MPPS] Using cached access token")
+
+            # Step 8b: Call set_tags with the token
+            headers = {
+                "Content-Type": "application/json",
+                "Authorization": f"Bearer {access_token}",
+            }
+
+            response = requests.post(
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)